src/zoom/iniZoomRestrict.js

```diff
@@ -6,17 +6,23 @@ export default (function iniZoomRestrict(max_zoom, labels, viz_dim) {
   const col_vs_row_space =
     num_col / viz_dim.heat.width / (num_row / viz_dim.heat.height);
   // working on improved matrix zooming
-  const zoom_restrict = {
-    x: {
-      min: 1.0,
-      max: num_row > num_col ? max_zoom * (1 / col_vs_row_space) : max_zoom,
-      ratio: num_row > num_col ? 1 / col_vs_row_space : 1.0,
-    },
-    y: {
-      min: 1.0,
-      max: num_row > num_col ? max_zoom * col_vs_row_space : max_zoom,
-      ratio: num_row > num_col ? col_vs_row_space : 1.0,
-    },
-  };
+  // used the code from clustergrammer repo for zooming in row or col direction
+  const zoom_restrict = {};
```

Comment:
A better approach in the comment would be to link the source repo's url here - since this repo is also clustergrammer

Reply:
Sure I will add the repo url